### PR TITLE
More groupable titles and other improvements

### PR DESCRIPTION
**Pull request by shtrom**

### CODE REVIEW

File: plugins/backup/fresh-backups [hunk 1]
```diff
@@ -39,18 +39,21 @@ An example configuration snippet for backup-manager [0] follows.
 
 =head1 AUTHOR
 
-Olivier Mehani <shtrom+munin@ssji.net>
+Copyright (C) 2016,2019 Olivier Mehani <shtrom+munin@ssji.net>
 
 =head1 LICENSE
 
-GPLv2
+SPDX-License-Identifier: GPL-3.0-or-later
```
Comment (shtrom) [inline plugins/backup/fresh-backups]: I'm the original author, and the only one to have introduced meaningful changes. I hope the relicensing is OK.

File: plugins/network/hostsdeny [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
 #
 # Plugin to monitor the number of hosts in /etc/hosts.deny
 # that are deined access to sshd
 #
 # Based on denyhosts plugin by tjansson (2009)
 #
 # Copyright (C) 2009 Kåre Hartvig Jensen (kaare.hartvig.jensen@gmail.com)
+# Copyright (C) 2019 Olivier Mehani <shtrom+munin@ssji.net>
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
```
Comment (shtrom) [inline plugins/network/hostsdeny]: Added based on the text of the original license below.

File: plugins/sickbeard/sickbeard [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+# Munin plugins for Sick-Beard
+#
+# Copyright (C) 2012 - Blauwbek
+# Copyright (C) 2012 - Thiago
+# Copyright (C) 2019 - Olivier Mehani <shtrom+munin@ssji.net>, for the
+# 		       multigraph support and better error handling
+#
+# Sick-Beard       : http://sickbeard.com/
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
```
Comment (shtrom) [inline plugins/sickbeard/sickbeard]: Based on the license text from the original plugins, and below.

**Comment by sumpfralle** [comment]
Thank you for this contribution!

They look good to me. Just for the new `sickbeard` plugin, I would appreciate a perldoc header. Would you mind changing this?

**Comment by shtrom** [comment]
Sure thing. Done.

**Comment by sumpfralle** [comment]
Great - thank you!

Just one last thing (I forgot to mention it before): the tests fail due to a plugin now being shellcheck-compliant (great!) and another one with an introduced shellcheck issue. Could you please take a look?

https://travis-ci.org/munin-monitoring/contrib/builds/596578180?utm_source=github_status&utm_medium=notification

**Comment by shtrom** [comment]
Fixed :+1:

**Comment by sumpfralle** [comment]
Great - thank you!

**Pull request merged**